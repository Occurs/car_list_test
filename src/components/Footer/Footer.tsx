import React from 'react';
import Grid from '@material-ui/core/Grid';
import Box from '@material-ui/core/Box';
import Typography from '@material-ui/core/Typography';
import Container from '@material-ui/core/Container';
import { colors } from 'styles/variables';

const Header = () => {
  return (
		<Box component='footer' height='80px' borderTop={`2px solid ${colors.secondary}`}>
			<Container maxWidth='md'>
				<Grid
					container
					direction='row'
					justify='center'
					alignItems='center'
				>
					<Grid item>
            <Typography>
              <Box component='span' height='80px' display='flex' alignItems='center' justifyContent='center' fontSize='12px'>
                Â© AUTO1 Group 2018
              </Box>
            </Typography>
					</Grid>
				</Grid>
			</Container>
		</Box>
  );
};

export default Header;
