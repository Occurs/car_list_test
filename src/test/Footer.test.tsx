
import React from "react";
import renderer from "react-test-renderer";
import Footer from "components/Footer/Footer";

describe("Footer Component", () => {
  test("snapshot", () => {
    const component = renderer.create(<Footer />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
