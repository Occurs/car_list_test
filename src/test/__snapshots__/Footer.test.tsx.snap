// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component snapshot 1`] = `
<footer
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthMd"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item"
      >
        <p
          className="MuiTypography-root MuiTypography-body1"
        >
          <span
            className="MuiBox-root MuiBox-root-2"
          >
            Â© AUTO1 Group 2018
          </span>
        </p>
      </div>
    </div>
  </div>
</footer>
`;
